src/chat/chatter.py

```diff
@@ -184,11 +184,25 @@ def update(self):
         url = client.instance.urls.get(player.login)
         if url:
             if url.scheme() == "fafgame":
-                self.statusItem.setIcon(util.icon("chat/status/lobby.png"))
-                self.statusItem.setToolTip("In Game Lobby<br/>"+url.toString())
+                game = client.instance.games.games[int(url.queryItemValue("uid"))]
```

Comment:
Same argument, it works, but it's really ugly. We need to get that uid from elsewhere.